pysat/instruments/cnofs_ivm.py

```diff
@@ -99,17 +98,12 @@ def clean(inst):
 
     Parameters
     -----------
-    inst : (pysat.Instrument)
+    inst : pysat.Instrument
```

Comment:
I can't find a way to insert a comment or suggestion on unchanged line.
for default method

```
"""Apply C/NOFS IVM default attributes

The sample rate for loaded data is attached at inst.sample_rate
before any attached custom methods are executed.

Parameters
----------
ivm : pysat.Instrument
    C/NOFS IVM Instrument object

 """

```

```suggestion
    inst : pysat.Instrument
```